lint-staged.config.js

```diff
@@ -1,11 +1,18 @@
 // @ts-check
 
+const commands = {
+  format: 'prettier --write',
+  /**
+   * Run eslint in fix mode for applicable files followed by prettier.
+   * - The eslint wrapper handles filtering which files should be linted, since we need to both:
+   *   - respect ignore files (which eslint doesn't do by default when passed a specific file path)
+   *   - match the set of files that are linted by the package's normal `lint` command
+   */
+  lint: 'node ./scripts/lint-staged/eslint',
+};
+
 // https://www.npmjs.com/package/lint-staged
 module.exports = {
-  // Run eslint in fix mode for applicable files followed by prettier.
-  // - The eslint wrapper handles filtering which files should be linted, since we need to both:
-  //   - respect ignore files (which eslint doesn't do by default when passed a specific file path)
-  //   - match the set of files that are linted by the package's normal `lint` command
-  // - Prettier must be run in sequence after eslint since both of them can modify files
-  '*': ['node ./scripts/lint-staged/eslint', 'prettier --write'],
+  '**/*.{js,jsx,ts,tsx,json,html,yml,css,scss,md,mdx}': [commands.format],
```

Comment:
@Hotell  - check that those wont clash

Reply:
Per the [package docs](https://www.npmjs.com/package/lint-staged#running-multiple-commands-in-a-sequence) I think eslint and prettier need to be run in sequence. `eslint --fix` may introduce formatting issues which would need to be fixed by prettier, and if they're in separate entries that wouldn't happen.

Reply:
done 846bbd659a2995ce943a29c09b29052b8115d269

I tested out this behaviour as its not documented. Now we will know ;) 

- looks like if there are identical keys in globs evaluated the first one wins.

![2021-08-13 at 10 37](https://user-images.githubusercontent.com/1223799/129329427-74c1987e-72a8-4e2b-98b6-18a038f8af2a.png)